plugin-core/src/main/java/appland/files/AppMapAsyncFileListener.java

```diff
@@ -40,17 +40,6 @@ public class AppMapAsyncFileListener implements AsyncFileListener {
     }
 
     private static boolean isAppMapFileChange(@NotNull VFileEvent event) {
-        if (event.isFromRefresh()) {
-            return false;
-        }
-
-        // only treat renames as valid property changes
```

Comment:
Is there a reason why you removed this?
A property change event could also be "encoding change" and others. With this code removed, these would be send as `Other` changes.

Reply:
Looks like I may have missed the negation. I'll add it back.